app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -295,18 +295,18 @@
             </label>
           </div>
         </div>
-        {{#if data.event.payByPayPal}}
+        {{#if data.event.canPayByPapal}}
           <div class="field">
             <label class="required">PayPal Email</label>
             {{input type='email'}}
           </div>
         {{/if}}
       {{/if}}
-      {{#if canAcceptStripe}}
```

Comment:
This is different. `canAcceptStripe` tells if the currency selected by user is supported by Stripe. So, this and the related computed property is needed.